pageserver/src/metrics.rs

```diff
@@ -112,6 +112,18 @@ static CURRENT_LOGICAL_SIZE: Lazy<UIntGaugeVec> = Lazy::new(|| {
     .expect("failed to define current logical size metric")
 });
 
+// Metrics collected on tenant statss.
+const TENANT_STATE_OPTIONS: &[&str] = &["loading", "attaching", "active", "stopping", "broken"];
```

Comment:
NIT: we can avoid possible duplication with `TenantState::as_str`.

Nice way to group constants in a module:
```rust
pub mod state {
    const LOADING: &str = "loading";
    // ...
}
```

So they can be referred to as `state::LOADING`.

Naming depends on where this module is placed.

Reply:
done